src/import/sst_service.rs

```diff
@@ -638,6 +688,10 @@ macro_rules! impl_write {
 
             let timer = Instant::now_coarse();
             let label = stringify!($fn);
+            if let Err(err) = self.check_deny() {
```

Comment:
What about only deny `ingest` and allow write/download since these request have no bad side-effect. In general, ingest is very fast so the retry cost is low but write&download is slow.

Reply:
Perhaps we can have a try.